plugin-build/src/main/kotlin/io/sentry/android/gradle/telemetry/SentryTelemetryService.kt

```diff
@@ -0,0 +1,474 @@
+            tags.put("SENTRY_uploadNativeSymbols", extension.uploadNativeSymbols.get().toString())
+            tags.put(
+                "SENTRY_autoUploadNativeSymbols",
+                extension.autoUploadNativeSymbols.get().toString()
+            )
+            tags.put("SENTRY_includeNativeSources", extension.includeNativeSources.get().toString())
+            // TODO PII?
+            tags.put(
+                "SENTRY_ignoredVariants_set",
+                extension.ignoredVariants.get().isNotEmpty().toString()
+            )
+            // TODO PII?
+            tags.put(
+                "SENTRY_ignoredBuildTypes_set",
+                extension.ignoredBuildTypes.get().isNotEmpty().toString()
+            )
+            // TODO PII?
+            tags.put(
+                "SENTRY_ignoredFlavors",
+                extension.ignoredFlavors.get().isNotEmpty().toString()
+            )
+            tags.put(
+                "SENTRY_experimentalGuardsquareSupport",
+                extension.experimentalGuardsquareSupport.get().toString()
+            )
+            tags.put(
+                "SENTRY_tracing_enabled",
+                extension.tracingInstrumentation.enabled.get().toString()
+            )
+            tags.put(
+                "SENTRY_tracing_debug",
+                extension.tracingInstrumentation.debug.get().toString()
+            )
+            tags.put(
+                "SENTRY_tracing_forceInstrumentDependencies",
+                extension.tracingInstrumentation.forceInstrumentDependencies.get().toString()
+            )
+            tags.put(
+                "SENTRY_tracing_features",
+                extension.tracingInstrumentation.features.get().toString()
+            )
+            tags.put(
+                "SENTRY_tracing_logcat_enabled",
+                extension.tracingInstrumentation.logcat.enabled.get().toString()
+            )
+            tags.put(
+                "SENTRY_tracing_logcat_minLevel",
+                extension.tracingInstrumentation.logcat.minLevel.get().toString()
+            )
+            tags.put(
+                "SENTRY_autoInstallation_enabled",
+                extension.autoInstallation.enabled.get().toString()
+            )
+            tags.put(
+                "SENTRY_autoInstallation_sentryVersion",
+                extension.autoInstallation.sentryVersion.get().toString()
+            )
+            tags.put(
+                "SENTRY_includeDependenciesReport",
+                extension.includeDependenciesReport.get().toString()
+            )
+            tags.put("SENTRY_includeSourceContext", extension.includeSourceContext.get().toString())
+            // TODO PII?
+            tags.put(
+                "SENTRY_additionalSourceDirsForSourceContext_set",
+                extension.additionalSourceDirsForSourceContext.get().isNotEmpty().toString()
+            )
+            // TODO PII?
+            extension.org.orNull?.let { tags.put("SENTRY_org", it) }
```

Comment:
do we need this as a tag, if we already set it as `user.id`?

Reply:
probably not